### DISCUSSION TITLE
Any plans for a new release?

### DISCUSSION THREAD

**Discussion by hakostra** [pyvista/pyvista#2755] [Ideas]
Excluding dependencies updates, there are [~3 pages worth of merged PR's](https://github.com/pyvista/pyvista/pulls?page=1&q=is%3Apr+is%3Aclosed+-label%3Adependencies) since last release in April...

Any plans for a new release soon?

**Comment by banesullivan** [comment]
Coming soon ;)

Please follow https://github.com/pyvista/pyvista/pull/2863
